concent_api/core/models.py

```diff
@@ -404,9 +406,40 @@ class RecipientType(ChoiceEnum):
         Requestor   = 'requestor'
 
     payment_ts              = DateTimeField()
-    task_owner_key          = Base64Field(max_length = 64)
-    provider_eth_account    = Base64Field(max_length = 64)
-    amount_paid             = DecimalField(max_digits = 10, decimal_places = 2)
+    task_owner_key          = CharField(max_length = CLIENT_DETAILS_LENGTH)
+    provider_eth_account    = CharField(max_length = ETHEREUM_ADDRESS_LENGTH)
+    amount_paid             = IntegerField()
     recipient_type          = CharField(max_length = 32, choices = RecipientType.choices())
-    amount_pending          = DecimalField(max_digits = 10, decimal_places = 2)
+    amount_pending          = IntegerField()
     pending_response        = ForeignKey(PendingResponse, related_name = 'payments')
+
+    def clean(self):
+        if self.task_owner_key == self.provider_eth_account:
+            raise ValidationError({
+                'provider_eth_account': 'Provider ethereum account address must be diffrent than task owner key'
+            })
+
+        if not isinstance(self.amount_pending, int) or self.amount_pending <= 0:
+            raise ValidationError({
+                'amount_pending': 'Amount pending must be an integer and bigger than or equal 0'
+            })
+
+        if not isinstance(self.amount_paid, int) or self.amount_paid < 0:
+            raise ValidationError({
+                'amount_paid': 'Amount paid must be an integer and bigger than or equal 0'
```

Comment:
Update error message.

Reply:
But here is good error message. Amount paid can be 0 and bigger than 0